### PR TITLE
Fix Python bindings installation and support building bindings as standalone projects

### PR DESCRIPTION
**Pull request by traversaro**
I initially just fixed the installation of `__init__.py` for `wearables` and fixed compilation on Windows, to permit to merge https://github.com/robotology/robotology-superbuild/pull/1717 (see https://github.com/robotology/human-dynamics-estimation/pull/398/commits/5e0ac8c33bb291cf4469c8f454ee4ee152796600, https://github.com/robotology/human-dynamics-estimation/pull/398/commits/72f02aed7e713e4ccbce09add231372c1d6cd751 and https://github.com/robotology/human-dynamics-estimation/pull/398/commits/7b42767d7dc87b39a58bc5c3a26c48a2b81f099e).

However, while I was at it, I thought it was useful to also fix https://github.com/robotology/robotology-superbuild/issues/1268, propagating what was done in https://github.com/robotology/idyntree/pull/1209 (https://github.com/robotology/human-dynamics-estimation/pull/398/commits/5ed3e382cefef0148ff9d26d8b6a38f6fbd22574).

Then, as this was necessary to package easily hde on conda-forge, I also added support to compile the python bindings as a standalone project, linking against an existing external HDE installation (https://github.com/robotology/human-dynamics-estimation/pull/398/commits/5b6d09f268438a561f03ca77831dc405530a74b4 and https://github.com/robotology/human-dynamics-estimation/pull/398/commits/002b75de1a7e97c913b7fc84df251345dce0132f).

This PR change the internal ALIAS of `WearableData` and `WearableActuators` target to `WearableData::WearableData` and `WearableActuators::WearableActuators` for consistency with the namespace used during the install.  If you have been using `wearables` via `FetchContent`/`add_subdirectory`, please change all your instances of `Wearable::WearableData` to `WearableData::WearableData` and of of `Wearable::WearableActuators` to `WearableActuators::WearableActuators`.

### CODE REVIEW

**Review by dariosortino — APPROVED** [review]
Consider this approval as an ack of these modifications, I'm not very familiar with cmake and bindings in general

**Comment by traversaro** [comment]
> Consider this approval as an ack of these modifications, I'm not very familiar with cmake and bindings in general

Yes, I think for most ami-iit's repos reviews are mostly an ack, unless the maintainer wants to go into deep and comments.

File: msgs/CMakeLists.txt
Comment (S-Dafarra) [inline msgs/CMakeLists.txt:63]: Did the alias change? This might be breaking in some private code (cc @dariosortino)

Comment (dariosortino) [reply msgs/CMakeLists.txt:63]: Tagging @davidegorbani  as he is more familiar in writing cmake for ifeel applications

Comment (traversaro) [reply msgs/CMakeLists.txt:63]: The alias is only used for code internal to the `wearables`/`human-dynamics-estimation`, or project that included them via `FetchContent`/`add_subdirectory` (none, if I am not wrong), so we should not risk any regression.

The reason why I changed is that before this PR (and the merge PR), this was the situation in wearables:

| Target name during `build` | Imported target name after install |
|:---------------------------:|:--------------------------------:|
| `Wearable::IWear`         |  `IWear::IWear` |
|  `Wearable::WearableData` | `WearableData::WearableData` |
| `Wearable::WearableActuators` | `WearableActuators ::WearableActuators` |

The first one was already changed in the merge commit, while the last two just in this PR.

The whole idea of the ALIAS targets is to be able to link to the same target in the code that consumes the library when built in the same project and the code that consumes the library via `find_package`, so using two different namespaces was basically making it useless. 

I can advertise better this change and if you are worried of somebody consuming the project via `add_subdirectory` I could even add back the `Wearable::` alias and deprecating them, but I am not sure the gains are worth the work, even if it is minimal.

Comment (S-Dafarra) [reply msgs/CMakeLists.txt:63]: Ah I see. Indeed, the math I found in the private code was using ``WearableActuators::WearableActuators``. No problem then!

**Review by S-Dafarra — APPROVED** [review]

File: msgs/CMakeLists.txt
Comment (traversaro) [reply msgs/CMakeLists.txt:63]: Just in case, I updated the PR messages in https://github.com/robotology/human-dynamics-estimation/pull/398 and https://github.com/robotology/human-dynamics-estimation/pull/395 to be explicit about this change.

**Pull request merged**
